orca/api/resources/v1/alert.py

```diff
@@ -0,0 +1,52 @@
+# Copyright 2020 OpenRCA Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from flask_restx import Model, Namespace, Resource, fields, marshal
+
+node_fields = Model('Alert Node', {
```

Comment:
`node_fields` is a left-over after the Graph API. Let's rename to `alert_fields`. We could also rename the `alerts_fields` (below) to `alert_list_fields`.